backend/ops_api/features/portfolio/calculate_portfolio_funding.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Calculate Portfolio Funding By Research Project
+
+  Scenario: Calculate FY Total Portfolio Budget For All 1-Year CANs
+  (Displayed on the "Projects and Spending" tab on the Portfolio details
+  page in the FY XXXX Budget vs Spending container.)
+    Given a set of CAN data with the current FY 2023
+      | CAN | total funding |
+      | 1   | $100          |
+      | 2   | $150          |
+      | 3   | $250          |
+    When I calculate the Total Portfolio Budget
+    Then the result should be $500.
+
+  Scenario: Calculate FY Total Portfolio Budget With Carry-Over CANs
```

Comment:
@johndeange So in the scenario above (lines 3-12), we're intending only to show the current FY's funding. But in this sencario, we're showing the aggregate funding? Is that the intent?  because the "Given" statement says "with the current FY 2023" and that threw me off.